apache-geode/src/main/java/com/baeldung/geode/Customer.java

```diff
@@ -0,0 +1,78 @@
+package com.baeldung.geode;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class Customer implements Serializable {
+
+    private static final long serialVersionUID = -7482516011038799900L;
+
+    private CustomerKey key;
+    private String firstName;
+    private String lastName;
+    private Integer age;
+
+    public Customer() {
+    }
+
+    public Customer(String firstName, String lastName, int age) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.age = age;
+    }
+
+    public Customer(CustomerKey key, String firstName, String lastName, int age) {
+        this(firstName, lastName, age);
+        this.key = key;
+    }
+
+    // setters and getters
+
+    public static long getSerialVersionUID() {
```

Comment:
Does Geode need this? Otherwise, let's take it out.

Reply:
I was getting runtime issues which were solved by implementing serialversionUID